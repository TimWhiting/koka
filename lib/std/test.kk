/*----------------------------------------------------------------------------
   Copyright 2024 Tim Whiting

   Licensed under the Apache License, Version 2.0 ("The License"). You may not
   use this file except in compliance with the License. A copy of the License
   can be found in the LICENSE file at the root of this distribution.
----------------------------------------------------------------------------*/
pub import test/test