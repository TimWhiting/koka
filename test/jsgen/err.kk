import std/os/env
import std/os/file
import std/os/path

fun main()
  val args = get-args()
  val file-path = match args
    Cons(file-path, Nil) -> file-path
    _ -> throw("Usage: program <file path>", ExnError)
  val file-contents = read-text-file(file-path.path)
  ()