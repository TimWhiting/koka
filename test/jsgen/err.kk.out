uncaught exception: Usage: program <file path>